### PR TITLE
Add UniFi Protect binary_sensor platform

### PR DESCRIPTION
**Pull request by AngellusMortis**
## Proposed change

Adds docs for the `binary_sensor` platform.

As with the previous PR, all of the new sensors are grouped as a single item to only mention their existence. This is in line with feedback and another related PR: https://github.com/home-assistant/home-assistant.io/pull/20990.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/63489
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/unifiprotect.markdown [hunk 1]
```diff
@@ -112,6 +114,7 @@ Each UniFi Protect camera will get the following entities added:
   * **Privacy Mode** - If your camera allows for Privacy Masks, there will be a configuration switch to toggle a "Privacy Mode" that disables recording, microphone, and a black privacy zone over the whole camera.
   * **System Sounds** - If your camera has a speaker, there will be a configuration switch to toggle system sounds.
   * **SSH Enabled** - A disabled by default switch entity will be added to let you toggle on and off SSH for your camera. Username is `ubnt` and password is your Device Password as configured in UniFi Protect.
+* **Sensors** - Sensors are provided for data provided from camera device. Sensors include "Is Dark" and "Doorbell Chime" (if camera has a chime).
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Sensors** - Sensors are provided for data provided from the camera device. Sensors include "Is Dark" and "Doorbell Chime" (if camera has a chime).
```

File: source/_integrations/unifiprotect.markdown [hunk 2]
```diff
@@ -125,12 +128,14 @@ Each UniFi Protect floodlight will get the following entities added:
 * **Switch** - The following switch entities will be added for each floodlight:
   * **Status Light** - A configuration switch to turn it on and off to turn on and off the status light for your floodlight.
   * **SSH Enabled** - A disabled by default switch entity will be added to let you toggle on and off SSH for your floodlight. Username is `ubnt` and password is your Device Password as configured in UniFi Protect.
+* **Sensors** - Sensors are provided for data provided from floodlight device. Sensors include "Is Dark" and "Motion Detected".
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Sensors** - Sensors are provided for data provided from floodlight devices. Sensors include "Is Dark" and "Motion Detected".
```

File: source/_integrations/unifiprotect.markdown [hunk 3]
```diff
@@ -125,12 +128,14 @@ Each UniFi Protect floodlight will get the following entities added:
...
 * **Button** - A disabled by default button entity is added for each floodlight device. The button will let you reboot your floodlight device.
 
 ### Sensors
 
 Each UniFi Protect smart sensor will get the following entities added:
 
+* **Sensors** - Sensors are provided for data provided from smart sensor device. Sensors include "Door", "Motion Detected" and "Battery Low".
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Sensors** - Sensors are provided for data provided from smart sensor devices. Sensors include "Door", "Motion Detected" and "Battery Low".
```

File: source/_integrations/unifiprotect.markdown [hunk 4]
```diff
@@ -142,6 +147,12 @@ Each UniFi Protect viewport will get the following entities added:
 * **Button** - A disabled by default button entity is added for each viewport device. The button will let you reboot your viewport device.
 * **Switch** - A disabled by default switch entity will be added to let you toggle on and off SSH for your viewport. Username is `ubnt` and password is your Device Password as configured in UniFi Protect.
 
+### NVR
+
+You main UniFi Protect NVR device also gets a number of diagnostics sensors that can be use for tracking the state of your UniFi Protect system:
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
Your main UniFi Protect NVR device also gets a number of diagnostics sensors that can be used for tracking the state of your UniFi Protect system:
```

File: source/_integrations/unifiprotect.markdown [hunk 5]
```diff
@@ -142,6 +147,12 @@ Each UniFi Protect viewport will get the following entities added:
...
+
+* **Disk Health**: Each disk installed in your NVR will have a disk health sensor. These are simple good/bad sensors and the order is not promised to match the order in UniFi OS. Disk model number is provided as an state attribute though to help map sensor to disk.
```
Comment (bdraco) [inline source/_integrations/unifiprotect.markdown]:
```suggestion
* **Disk Health**: Each disk installed in your NVR will have a disk health sensor. These are simple good/bad sensors and the order is not promised to match the order in UniFi OS. Disk model number is provided as a state attribute though to help map sensor to disk.
```

**Review by bdraco — APPROVED** [review]

**Pull request merged**